### DISCUSSION TITLE
How to add favicon in vitepress ?

### DISCUSSION THREAD

**Discussion by iqlal** [vuejs/vitepress#2475] [Q&A]
How to add favicon in vitepress ?

**Comment by userquin** [comment]
Don't use this: ~~https://github.com/iconify/website/blob/main/.vitepress/config.ts#L35~~

I'll fix previous code, use this: https://github.com/vite-pwa/docs/blob/main/.vitepress/config.ts#L255-L256

Check the docs https://vitepress.dev/reference/site-config#head

**Reply by alephpi** [reply]
It means https://github.com/vite-pwa/docs/blob/3b8808e6e46683dac49e50318ecf494085ba970c/.vitepress/config.ts#L256 for those who come later.

**Reply by userquin** [reply]
upps, thx @alephpi , a few changes with latest releases in VitePWA repo

**Reply by alephpi** [reply]
Hi, BTW I just wonder why the favicon on my page isn't rendered. Here's a screenshot, kind of weird...
The same svg can be rendered in NavBar but not as favicon...
![image](https://github.com/vuejs/vitepress/assets/61275421/d32d88fc-ff3c-437f-b816-1e077e031656)

**Reply by userquin** [reply]
can you test it using private browsing, maybe browser cache? can you test it in another browser (Chrome/FF) ?

**Reply by alephpi** [reply]
indeed, thx!

**Comment by dddddgz05** [comment]
Edit `config.mts`:
```
...
export default defineConfig({
  title: "*your title*",
  description: "*your description*",
  head: [['link', { rel: 'icon', href: '/favicon.ico' }]],
...
```
Then add your icon at `docs/public/favicon.ico`, run `docs:dev` command, open `localhost:5173`.

If your favicon still invisible, press `Ctrl+F5` to refresh the cache.
